types/xdg_shell/wlr_xdg_surface.c

```diff
@@ -81,14 +81,13 @@ void unmap_xdg_surface(struct wlr_xdg_surface *surface) {
 	}
 
 	surface->configured = surface->mapped = false;
-	surface->configure_serial = 0;
 	if (surface->configure_idle) {
 		wl_event_source_remove(surface->configure_idle);
 		surface->configure_idle = NULL;
 	}
 	surface->configure_next_serial = 0;
```

Comment:
With `surface->configure_serial = 0` removed, this should be removed as well.

Reply:
Hm, I'm not sure. `surface->configure_serial = 0` is removed because the `memset` below takes care of resetting it in `current`.